source/localizable/configuring-ember/disabling-prototype-extensions.md

```diff
@@ -92,7 +92,7 @@ islands.contains('Oahu');
 ### Strings
 
 Strings will no longer have the convenience methods described in the
-[Ember.String API reference.](http://emberjs.com/api/classes/Ember.String.html). Instead,
+[Ember.String API reference](http://emberjs.com/api/classes/Ember.String.html). Instead,
```

Comment:
While you're in here, can you put backticks in `Ember.String`, and a newline after the period please?